src/index.html

```diff
@@ -5,9 +5,118 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="./images/favicon.png">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,400,400i,500,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="logo header__logo"></div>
+        <div class="header__phone">+1 234 555-55-55</div>
+        <div class="header__action"><a href="" class="header__link">call to order</a></div>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item_active">Home</li>
+          <li class="nav__item"><a href="#" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Gallery</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Consultation</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Testimonials</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="section">
+        <div class="info">
+          <h1 class="info__heading">Miami <span class="bold-text">best</span> real estate <a href="#" class="link">here</a></h1>
+          <div class="info__text">
+            <p><span class="bold-text">Over 50 years experience in love.</span><br>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+            <p>The market leader in real estate since 2003.<br>Awards Real Estate Company, 2010, 2011, 2013.<br>More than 200 satisfied customers premium segment.<br>We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information about the profitable deals.</p>
+            <p class="bold-text">We will never be late!<br>Guaranteed secure transaction - 100%.</p>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <h2 class="heading section__heading">Gallery<span class="heading__crown"></span></h2>
+        <div class="gallery">
+          <div class="photos">
+            <img src="images/photos/1.jpg" alt="First photo from gallery" class="photos__img photos_wide">
```

Comment:
длинные теги можно также переносить, иначе твой код нечитаем
```suggestion
            <img
              src="images/photos/1.jpg" 
              alt="First photo from gallery" 
              class="photos__img photos_wide"
            >
```